Frends.MongoDB.Insert/Frends.MongoDB.Insert.Tests/Files/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+﻿version: '3'
+services:
+  mongo:
+    container_name: mongo
+    image: mongo:latest
```

Comment:
Maybe you should use a specific version of the image, so that we can be sure that it will work in the future too. Using `latest` means that we will always take the latest image, so if there are new versions, we will then be using those ones, which means it is possible that a change to the image will break the tests.